recipes/pyunlocbox/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyunlocbox" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "50e6e900fe321e6335a6cf71854156075650068bd75980a33f1ae3b83215f5ca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pyunlocbox
+    - pyunlocbox.tests
+
+about:
+  home: https://github.com/epfl-lts2/pyunlocbox
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Please manually package the `license_file` and raise an issue in the source repo asking for it to be included in future source distributions. Add a link to the issue as a comment in the recipe:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
I'm managing the source repo and thought about including the license file in the next release. But I'd rather not make a release just for that. As I think the BSD does not require a license file in the package, would it be acceptable to have none until the next release?